pallets/asset-handler/src/tests.rs

```diff
@@ -291,6 +291,15 @@ pub fn test_set_block_delay() {
 	});
 }
 
+#[test]
+pub fn test_account_balances() {
+	new_test_ext().execute_with(|| {
+		let balances_vec = AssetHandler::account_balances(vec![1], 1_u64);
+		assert_eq!(balances_vec.len(), 1);
+		assert_eq!(balances_vec[0], 0_u128);
+	});
+}
+
```

Comment:
I think should be added test in dynamic (e.g. increasing balance and after that request to receive balance and assert that value changed...).

Reply:
Updated ✅